networkit/cpp/edgescores/EdgeScoreLinearizer.cpp

```diff
@@ -47,7 +47,7 @@ void EdgeScoreLinearizer::run() {
         for (omp_index pos = 0; pos < static_cast<omp_index>(G->upperEdgeIdBound()); ++pos) {
             edgeid eid = std::get<2>(sorted[pos]);
             if (eid != none) {
-                scoreData[eid] = pos * 1.0 / (G->numberOfEdges()-1); // TODO maybe writing back to sorted and then sorting again by eid could be faster (cache efficiency)?
+                scoreData[eid] = pos * 1.0 / (G->numberOfEdges()-1.0); // TODO maybe writing back to sorted and then sorting again by eid could be faster (cache efficiency)?
```

Comment:
A static_cast for `pos` and the denominator is needed + maybe a wrap (uint64 -> double).